docusaurus.config.js

```diff
@@ -82,13 +82,41 @@ module.exports = {
               href: "https://github.com/home-assistant",
             },
             {
-              label: "Twitter",
-              href: "https://twitter.com/hass_devs",
+              label: "X (Twitter)",
+              href: "https://x.com/hass_devs",
```

Comment:
@frenck this seems to not be active should we keep the link?

Reply:
Yes, it is active.